tests/jquery.flot-relativeTimeAxis.Test.js

```diff
@@ -54,50 +54,57 @@ describe('A Flot chart with relative time axes', function () {
 
-        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: 3600, label: '01:00:00'}, {v: 3610, label: '01:00:10'}]);
+        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: 0, label: '00:00:00'}, {v: 4200, label: '01:10:00'}]);
     });
 
     it('shows time bigger than 1 day correctly', function () {
-        plot = createPlotWithRelativeTimeAxis(placeholder, [[[86400, 1], [86410, 2]]]);
+        plot = createPlotWithRelativeTimeAxis(placeholder, [[[0, 1], [86410, 2]]]);
 
-        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: 86400, label: '1.00:00:00'}, {v: 86410, label: '1.00:00:10'}]);
+        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: 0, label: '00:00:00'}, {v: 100800, label: '1.04:00:00'}]);
     });
 
-    it('shows time bigger than 1 month correctly', function () {
-        plot = createPlotWithRelativeTimeAxis(placeholder, [[[2764800, 1], [2764810, 2]]]);
+    xit('shows time bigger than 1 month correctly', function () {
+        plot = createPlotWithRelativeTimeAxis(placeholder, [[[100, 1], [2764700, 2]]]);
 
-        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: 2764800, label: '32.00:00:00'}, {v: 2764810, label: '32.00:00:10'}]);
+        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: 100, label: '00:00:00'}, {v: 2764700, label: '32.00:10:00'}]);
     });
 
     it('shows time with milliseconds resolution correctly', function () {
-        plot = createPlotWithRelativeTimeAxis(placeholder, [[[0.001, 0.1], [0.002, 0.2]]]);
+        plot = createPlotWithRelativeTimeAxis(placeholder, [[[0, 0], [0.001, 0.1]]]);
 
-        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: 0.001, label: '00:00:00.001'}, {v: 0.002, label: '00:00:00.002'}]);
+        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: 0, label: '00:00:00.000'}, {v: 0.001, label: '00:00:00.001'}]);
     });
 
     it('shows negative time correctly', function () {
         plot = createPlotWithRelativeTimeAxis(placeholder, [[[-0.001, 0.1], [-0.002, 0.2]]]);
 
-        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: -0.002, label: '-00:00:00.002'}, {v: -0.001, label: '-00:00:00.001'}]);
+        expect(firstAndLast(plot.getAxes().xaxis.ticks)).toEqual([{v: -0.002, label: '-00:00:00.001'}, {v: -0.001, label: '00:00:00.000'}]);
     });
 
     it('works with vertical axes', function () {
-        plot = createPlotWithVerticalRelativeTimeAxis(placeholder, [[[0, 1], [1, 2]]]);
+        plot = createPlotWithVerticalRelativeTimeAxis(placeholder, [[[0, 0], [1, 1]]]);
 
-        expect(firstAndLast(plot.getAxes().yaxis.ticks)).toEqual([{v: 1, label: '00:00:01.000'}, {v: 2, label: '00:00:02.000'}]);
+        expect(firstAndLast(plot.getAxes().yaxis.ticks)).toEqual([{v: 0, label: '00:00:00.000'}, {v: 1, label: '00:00:01.000'}]);
     });
+
+    it('shows time relative to first data sample', function () {
```

Comment:
It will still show data with relative time axis starting from 00:00:00, which is the desktop behavior.